solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -70,19 +70,19 @@ hostname=cortx-deploy-node2.cortx.com,user=root,pass=<root-password>
 hostname=cortx-deploy-node3.cortx.com,user=root,pass=<root-password>
 ```
 
--  Execute `cluster-setup.sh` to setup K8s cluster on your VM for N-node deployment.
--  To allow PODs creation on primary node pass the first input parameter for `cluster-setup.sh` script as true. Please note you must pass the input parameter as true for Single Node Setup.
+-  Execute `cluster-setup.sh` script from primary node to setup K8s cluster on your EC2 instances for N-node deployment.
+-  To allow PODs creation on primary node to pass the first input parameter for `cluster-setup.sh` script as true. Please note you must pass the input parameter as true for N-node setup.
 
 ```
-# ./cluster-setup.sh true
+./cluster-setup.sh true
 ```
 
 ## Deploy CORTX Stack 
 
-Execute `cortx-deploy.sh` to deploy the CORTX on your K8s Cluster.
+Execute `cortx-deploy.sh` script from primary node to deploy the CORTX stack on your K8s cluster.
```

Comment:
We will need instructions to identify a primary node. Please add any N Node related details in https://github.com/Seagate/cortx-re/tree/f-94l-multinode

Reply:
Its addressed in https://github.com/Seagate/cortx-re/pull/1156